### PR TITLE
Feat:added playlist functionality

### PR DESCRIPTION
**Pull request by talrejah02**
Feature added
- playlist functionality
alerts and active styles are yet to be added
please review and give suggestions.

### CODE REVIEW

File: src/components/navbar/nav.jsx [hunk 1]
```diff
@@ -7,7 +7,6 @@ import history from "../../assets/svg/History.svg";
 import playlist from "../../assets/svg/playlist.svg";
 import Logo from "../../assets/svg/Logo.svg";
 import account from "../../assets/svg/account.svg";
-import Setting from "../../assets/svg/Setting.svg";
 import watchlater from "../../assets/svg/watchlater.svg";
 import "./nav.css";
```
Comment (pankaj-prog) [inline src/components/navbar/nav.jsx:11]: suggestion: you can use absolute imports

File: src/components/playlistCard/playlistCard.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  }
+  return (
+      <div className='card-horizontal' onClick={playlistCardHandler} >
+          <section className="horizontal-card-img">
+              <img src='Images/videoImages/sickness1.webp' />
+          </section>
+          <section className='horizontal-card-description'>
+        <span className='horizontal-card-title'>{playlist.title}</span>
+          <span className="card-svg" onClick={()=>deletePlaylisthandler(playlist._id)} >
```
Comment (pankaj-prog) [inline src/components/playlistCard/playlistCard.jsx:25]: use buttons for actions

**Review by pankaj-prog — APPROVED** [review]
Added some minor changes, 
other than that the variable naming is good, and also the code is well structured

**Review by muskaanshah — APPROVED** [review]

**Review by simran61 — APPROVED** [review]
Looks good
Nice folder structure.

**Review by ScoobyDubeyDo — APPROVED** [review]

**Pull request merged**